run_app.sh

```diff
@@ -28,7 +28,9 @@ else
   echo "Found ACG_CONFIG - RUNNING WITH CLOWDER"
   echo "RUNNING SYSTEM BASELINE SERVICE"
   PORT=8000
+  METRICS_PORT=9000
   APP_CONFIG='gunicorn.conf.py'
-  bash -c "FLASK_APP=system_baseline.app:get_flask_app_with_migration flask db upgrade"
-  exec gunicorn wsgi --bind=0.0.0.0:$PORT --access-logfile=- --config "$APP_CONFIG"
+  bash -c "if FLASK_APP=system_baseline.app:get_flask_app_with_migration flask db upgrade; then exit 0; else exit 1; fi"
```

Comment:
Is the `bash -c` needed? Can it be run directly?

Reply:
It's not  💯 

Thanks!